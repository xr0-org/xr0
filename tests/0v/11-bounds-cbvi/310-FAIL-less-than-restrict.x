#ifdef XR0

void
foo(int i) ~ [ setup: i = [0?5]; ];

#endif

void
foo(int i)
{
	int arr[2];
	if (i < 3) {
		arr[i] = 0;
	}
}
