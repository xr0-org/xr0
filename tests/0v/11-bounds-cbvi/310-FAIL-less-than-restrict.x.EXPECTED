11-bounds-cbvi/310-FAIL-less-than-restrict.x:12:3: out of bounds (foo | #0 âˆˆ {0?3})
